group_vars/all/logrotate.yml

```diff
@@ -17,4 +17,4 @@ logrotate_scripts:
         if [ -d /etc/logrotate.d/httpd-prerotate ]; then \
               run-parts /etc/logrotate.d/httpd-prerotate; \
             fi \
-      postrotate: invoke-rc.d nginx rotate >/dev/null 2>&1
+      postrotate: "[ -f /var/run/nginx.pid ] && kill -USR1 `cat /var/run/nginx.pid`"
```

Comment:
Or..... I can ask for input. `strace` does indicate a performance benefit to `read pid < file` versus `cat file`. However, after trying everything I can think of to get the command working inside `/etc/logrotate.d/wordpress-sites`, I'm stuck with variations along the lines of this...

```
    PIDfile="/var/run/nginx.pid"
    if test ! -f $PIDfile
    then
      read nginxPID < $PIDfile
      kill -USR1 $nginxPID
    fi
```
or other similar. All attempts I've tried to get `$(<"/var/run/nginx.pid")` working with the `kill` command haven't found success. Not sure if it's an issue with syntax or what, but only works if I split it into two lines (`read` then `kill`) and explicitly use `read`. Depending on what I've tried I get either nothing at all (no error/no successful kill command) or permission denied errors.

Any thoughts?